.gitignore

```diff
@@ -1,6 +1,6 @@
-.DS_Store
-coverage
-vendor
-Resources/doc/_build
+/build
+/Resources/doc/_build
+/vendor
 composer.lock
+phpunit.xml
 .php_cs.cache
```

Comment:
Please sort the files as well like in https://github.com/sonata-project/SonataUserBundle/blob/3.x/.gitignore